wiki-webui/src/main/java/org/exoplatform/wiki/commons/Utils.java

```diff
@@ -255,7 +254,6 @@ public static void feedDataForWYSIWYGEditor(UIWikiPageEditForm pageEditForm, Str
       markup = (markupInput.getValue() == null) ? "" : markupInput.getValue();
     }
     String xhtmlContent = renderingService.render(markup, markupSyntax, Syntax.ANNOTATED_XHTML_1_0.toIdString(), false);
-    xhtmlContent = StringCommonUtils.encodeScriptMarkup(xhtmlContent);

```

Comment:
Don't delete the sanitizer, but improve it to fix the problem and the original XSS issue.